### DISCUSSION TITLE
Is this version on master branch compatible with the latest prisma?

### DISCUSSION THREAD

**Discussion by Cyrusky** [omar-dulaimi/prisma-joi-generator#2] [Q&A] [answered]
Just like the title said, I am wondering if it is compatible?
when I execute `npm install` or `yarn` or even `pnpm install` I got this warning.

```
warning " > prisma-joi-generator@0.0.0-rc.11" has incorrect peer dependency "@prisma/client@~3.12.0".
warning " > prisma-joi-generator@0.0.0-rc.11" has incorrect peer dependency "prisma@~3.12.0".
warning " > prisma-joi-generator@0.0.0-rc.11" has unmet peer dependency "tslib@^2.3.1".
```

**Comment by omar-dulaimi** [comment, marked as answer]
Hey @Cyrusky 

Thank you for letting me know. The issue is now fixed in `prisma-joi-generator@0.1.0`.

Also, don't forget to star the project :)

**Comment by Cyrusky** [comment]
Thx a lot for your effort, but I have a little suggestion. Could you make the file path shorten, the import statement is so loooooong, seems not very elegant. And files number are also too many. Maybe you can combine some files with similar functions.

**Reply by Cyrusky** [reply]
<img width="1025" alt="image" src="https://user-images.githubusercontent.com/13082989/170806877-b6ca1420-3647-4f73-9ec8-c12eb2fe419d.png">

**Comment by omar-dulaimi** [comment]
Hey @Cyrusky 

Glad it works now for you.

Could you instead open a feature request?
I appreciate it!
